tools/repack-kernel

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+# Author: Maciej Borzecki
+# Modified by Ian Johnson <ian.johnson@canonical.com>
+
+set -e
+
+if [[ -n "$D" ]]; then
+    set -x
+fi
+
+show_help() {
+    echo "usage: repack-kernel <command> <opts>"
+    echo ""
+    echo "handle extraction of the kernel snap to a workspace directory, and later"
+    echo "repacking it back to a snap."
+    echo ""
+    echo "   setup                          - setup system dependencies"
+    echo "   extract <snap-file> <target>   - extract under <target> workspace tree"
```

Comment:
could you include repack-kernel in the commands so then it is easier to copy&paste, like:
echo "  repack-kernel extract <snap-file> <target> "